src/operator_repo/checks/__init__.py

```diff
@@ -145,4 +175,6 @@ def run_suite(
         for target_type_name, target_type in SUPPORTED_TYPES:
             if isinstance(target, target_type):
                 for check in checks.get(target_type_name, []):
-                    yield from run_check(check, target)
+                    if tests and check.__name__ not in tests:
+                        continue
+                    yield from run_check(check, target, suite_name)
```

Comment:
This will only allow to pick a subset of the checks in a given suite.
For the purpose of ISV-5287, instead of changing the behaviour of run_suite, I'd rather add a function that takes a list of fully qualified check names and executes them, and I think it would make more sense to do that in the validation script rather than in this library.